sdks/java/core/build.gradle

```diff
@@ -24,6 +24,19 @@ applyJavaNature(
     'FileIO': 'https://github.com/typetools/checker-framework/issues/6388',
     'MergingActiveWindowSetTest': 'https://github.com/typetools/checker-framework/issues/3776',
     'WindowFnTestUtils': 'https://github.com/typetools/checker-framework/issues/3776',
+    'WindowIntoTranslation': 'https://github.com/typetools/checker-framework/issues/3791',
```

Comment:
https://github.com/typetools/checker-framework/issues/3791 has been fixed.

Should we reference a different bug here ?

Reply:
I did a checkerframework upgrade but didn't update this list. Seeing if these no longer crash now.